pkg/api/v1/types.go

```diff
@@ -1168,6 +1168,12 @@ type ContainerStatus struct {
 	// Container's ID in the format 'docker://<container_id>'.
 	// More info: http://releases.k8s.io/HEAD/docs/user-guide/container-environment.md#container-information
 	ContainerID string `json:"containerID,omitempty"`
+	// The image creation time from docker image inspection
+	ImageCreationTime unversioned.Time `json:"imageCreationTime,omitempty"`
+	// The image size time from docker image inspection
```

Comment:
Typo.

Image size seems generally useful.

Note that we may support other types of images (e.g., ACI, compressed tarballs fetched via HTTPS) in the future.